src/MovieCard.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import './MovieCard.css';
+
+export default function MovieCard({ img, title, id}) {
```

Comment:
Might be worth making sure the export default is fine to be here and not at the end of the file.
I prefer this way too, just want to make sure it lines up w/ best practices and such